4679 add standard report translations docs (#81)
msupply-foundation/msupply_docs: content/docs/getting_started/report.md

![export button](/docs/getting_started/images/export_button.png)

The report will be downloaded as an excel file.

### Standard report translations

Standard reports will be translated to the user's language if available when generating the report.

> If a translation in the user's language is unavailable, an english translation will be used as a default.

Reviewer: ```suggestion
English will be used by default in places where translations in the user's language is unavailable.
```

Author: Nice Roxy  thank you. I'm guessing we have a active sentence structure preference for docs?

I'll add these in future :)
